lib/ifconfig/sunos/ifconfig.rb

```diff
@@ -31,7 +31,8 @@ def initialize(input=nil,netstat=nil,verbose=nil)
         when /\s+ether\s+/im
           @ifaces[iface_name] = EthernetAdapter.new(iface_name,iface,netstat)
         else
-          puts "Unknown Adapter Type: #{iface}" if @verbose
+          # fallback for virtual nics like bge:0
+          @ifaces[iface_name] = EthernetAdapter.new(iface_name,iface,netstat)
```

Comment:
The ROUTER flag makes me suspicious.  On Linux or FreeBSD, an alias is simply a second IP address assigned to the same interface.  They don't show up as an additional interface.  This looks more like a tun interface, which has a layer 3 address but no layer 2 address.  Can you point me to any documentation on Solaris virtual nics?

Once we figure out what this thing is, I would prefer to add a new NetworkInterface child class for it.